pysat/_instrument.py

```diff
@@ -3785,6 +3793,20 @@ def _kwargs_keys_to_func_name(kwargs_key):
     return func_name
 
 
+def _return_reserved_keywords():
+    """Return list of pysat reserved keywords
+
+    Returns
+    -------
+    out_list : list
+       List of strings for reserved keywords
+
+    """
+
+    return ['fnames', 'inst_id', 'tag', 'date_array', 'data_path',
+            'format_str', 'supported_tags', 'start', 'stop', 'freq']
```

Comment:
Would it not be simpler to just define this as a hidden attribute?

Reply:
I defined it as a hidden attribute at _instrument.py level. A copy is attached to each Instrument object as `inst.kwargs_reserved` to go along with `inst.kwargs` and `inst.kwargs_supported`